ampd/src/sui/verifier.rs

```diff
@@ -133,20 +73,21 @@ impl PartialEq<&VerifierSetConfirmation> for &SuiEvent {
             .chain(expected.created_at.to_be_bytes())
```

Comment:
Use sort_by for expected_signers to be explicit?

Reply:
Why though? For the context of this method, it doesn't care about the "correct" order nor does it know it, it only wants to compare equality of two vectors.